flink-python/src/test/java/org/apache/flink/client/python/PythonDependencyUtilsTest.java

```diff
@@ -0,0 +1,246 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.client.python;
+
+import org.apache.flink.api.common.cache.DistributedCache;
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.python.PythonOptions;
+import org.apache.flink.python.util.PythonDependencyUtils;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import static org.apache.flink.python.util.PythonDependencyUtils.PYTHON_ARCHIVES;
+import static org.apache.flink.python.util.PythonDependencyUtils.PYTHON_FILES;
+import static org.apache.flink.python.util.PythonDependencyUtils.PYTHON_REQUIREMENTS_FILE;
+
+/**
+ * Tests for PythonDependencyManager.
+ */
+public class PythonDependencyUtilsTest {
+
+	private List<Tuple2<String, DistributedCache.DistributedCacheEntry>> cachedFiles;
+
+	@Before
+	public void setUp() {
+		cachedFiles = new ArrayList<>();
+	}
+
+	@Test
+	public void testAddPythonFile() {
```

Comment:
Rename to `testPythonFiles` as we are testing the configuration python.files instead of the method `addPythonFile`. Same for the other test cases.